aws/sagemaker-python-sdk
# fix: fix repack pipeline step by putting inference.py in "code" sub dir

The repack pipeline step was putting the custom inference script in the top level dir, which is not put in the python moudle path by default. This change puts the inference script in the "code" directory which is added to python module path by default.

#### Testing Done
1. create project from abalone sample
2. modify pipeline to use entry point
3. modify sdk to copy sdk to copy entry point to `code` dir in the repack step
4.  `python setup.py sdist`
5. in studio `pip install sagemaker-2.39.2.dev0.tar.gz`
6. upsert and execute pipeline
7. verify successful inference - code pipeline passes and inference.py in code dir

#### General

- [x] I have read the [CONTRIBUTING](https://github.com/aws/sagemaker-python-sdk/blob/master/CONTRIBUTING.md) doc
- [x] I used the commit message format described in [CONTRIBUTING](https://github.com/aws/sagemaker-python-sdk/blob/master/CONTRIBUTING.md#committing-your-change)
- [x] I have passed the region in to all S3 and STS clients that I've initialized as part of this change.
- [x] I have updated any necessary documentation, including [READMEs](https://github.com/aws/sagemaker-python-sdk/blob/master/README.rst) and [API docs](https://github.com/aws/sagemaker-python-sdk/tree/master/doc) (if appropriate)

#### Tests

- [x] I have checked that my tests are not configured for a specific region or account (if appropriate)
- [x] I have used [`unique_name_from_base`](https://github.com/aws/sagemaker-python-sdk/blob/master/src/sagemaker/utils.py#L77) to create resource names in integ tests (if appropriate)

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.
